installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -0,0 +1,116 @@
+[id="installing-gcp-user-infra-vpc"]
+= Installing a cluster on GCP using Deployment Manager templates
+include::modules/common-attributes.adoc[]
+:context: installing-gcp-user-infra-vpc
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Google Cloud Platform (GCP) that uses infrastructure that you provide.
+
+The steps for performing a user-provided infrastructure install are outlined here. Several
```

Comment:
Again, shared vpc is missing. I suggest to update it to The steps for performing a user-provided infrastructure install into a shared VPC are outlined here